examples/todo/package.json

```diff
@@ -51,9 +51,12 @@
     "@loopback/http-caching-proxy": "^1.0.5",
     "@loopback/testlab": "^1.0.5",
     "@loopback/tslint-config": "^2.0.0",
+    "@types/express": "^4.16.1",
     "@types/lodash": "^4.14.109",
     "@types/node": "^10.11.2",
     "lodash": "^4.17.10",
+    "p-event": "^2.2.0",
+    "path": "^0.12.7",
```

Comment:
There is a built-in module named `path` from node.js itself. Is it intentional?

Reply:
Oh no, you're right.